src/events/pg.js

```diff
@@ -1,10 +1,11 @@
 const uuid4 = require('uuid4');
 const shimmer = require('shimmer');
+const tryRequire = require('try-require');
 /* eslint-disable import/no-unresolved, import/no-extraneous-dependencies */
-const pg = require('pg');
-const Pool = require('pg-pool');
-const sqlParser = require('node-sqlparser');
+const pg = tryRequire('pg');
+const Pool = tryRequire('pg-pool');
 /* eslint-enable import/no-unresolved, import/no-extraneous-dependencies */
+const sqlParser = require('node-sqlparser');
```

Comment:
Remove the eslint line above

Reply:
The eslint line above is `enable` (comes after the `disable`)

Reply:
👍